src/repos/ArticleRepo.ts

```diff
@@ -96,21 +97,6 @@ const getArticlesByPublicationID = async (
     });
 };
 
-const getArticlesByPublicationIDs = async (
-  publicationIDs: string[],
-  limit: number = DEFAULT_LIMIT,
-  offset: number = DEFAULT_OFFSET,
-): Promise<Article[]> => {
-  const uniquePubIDs = [...new Set(publicationIDs)].map((id) => new ObjectId(id));
-  console.log(uniquePubIDs);
-  const pubSlugs = await PublicationModel.find({ _id: { $in: uniquePubIDs } }).select('slug');
-  return getArticlesByPublicationSlugs(
-    pubSlugs.map((pub) => pub.slug),
-    limit,
-    offset,
-  );
-};
```

Comment:
Are we 100% sure the existing iOS app does not make use of this?

Reply:
Will readd, I thought it was removed in main